mysql-test/mroonga/storage/drop/table/t/have_foreign_key.test

```diff
@@ -0,0 +1,45 @@
+# Copyright(C) 2021  Horimoto Yasuhiro <horimoto@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+--source ../../../../include/mroonga/have_mroonga.inc
+
+SET NAMES UTF8MB4;
+
+--disable_warnings
+DROP DATABASE IF EXISTS test;
```

Comment:
Why should we need to recreate `test` database?

Reply:
I've made a mistake.
I removed recreation `test` database.